lib/perl/Genome/Config/AnalysisProject/Command/CopyConfig.t

```diff
@@ -51,6 +54,7 @@ my $target_project = Genome::Config::AnalysisProject->create(
 my $cmd = $class->create(
     to_project => $target_project,
     from_project => $source_project,
+    tags => 1,
```

Comment:
I'd skipped it, since there's no code that only runs in the "no tags" case, but it is worthwhile to see that the flag is respected.  I'll add some more tests!